package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@bcgsc-pori/graphkb-client",
   "version": "4.2.5",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
Is moving toward lockfileVersion v3 (npm v9) intentional and the way to go from now on with pori repos?
Note: backward compatible with npm v8 and node v16.13.0, but node v16 is shipped with npm v8, which generate lockfileVersion v2. I guess I could keep node v16 (like the GH checks) but upgrate to npm v9 for that specific repo? So I won't be generating lockfileVersion v2 if I have to update the package-lock.json.

Reply:
Ah I didn't notice this change being made. I'm developing with node 20/npm 10. Are you stuck using a lower node version for other repos?

Reply:
@elewis2 , I don't know if I'm 'stuck', lol. GH checks for GKB API are node v16-18-20, but for GKB loader, schema & parser it's v12-14-16; so v16 was some sort of a common ground. I don't mind moving everything to v16+. In any case, should we be developping with the earliest of the latest version of the stack? Right now, if I'm using Node v16 with npm v8, it will generate lockfileVersion v2 everytime I'll update the dependencies. I can probably make npm v9 my default while still using node v16, but actual client checks are done with node v16 and npm v8: https://github.com/bcgsc/pori_graphkb_client/actions/runs/8942499994/job/24565218427#step:4:9